practice-project-ui/src/Calculator.tsx

```diff
@@ -1,25 +1,63 @@
 import * as React from "react";
+// import axios from 'axios';
+// import dotenv from 'dotenv';
+
+// dotenv.config();
+
+// const baseURL = process.env.BASE_URL;
+
+enum Operations {
+  Add = "+",
+  Subtract = "-",
+  Multiply = "*",
+  Divide = "/"
+};
 
 function App() {
 
-  const [data, setData] = React.useState(null);
+  // const [data, setData] = React.useState(null);
 
-  React.useEffect(() => {
-    fetch("/api")
-      .then((res) => res.json())
-      .then((data) => setData(data.message));
-  }, []);
+  // async function calculate(event: React.FormEvent<HTMLFormElement>) {
+    
+  //   event.preventDefault();
+
+  //   try {
+  //     axios.post(`${baseURL}/calculate`, {
+  //       firstValue: (event.target as HTMLFormElement).firstValue.value,
+  //       operation: (event.target as HTMLFormElement).operation.value,
+  //       secondValue: (event.target as HTMLFormElement).secondValue.value
+  //     })
+  //     .then((response) => {
+  //       setData(response.data);
+  //     });
+  //   } catch (error) {
+  //     console.log(error.response);
+  //   }
+  // }
 
   return (
     <div>
       <header>
         <p>Calculator Component</p>
-        <p>{data}</p>
+        {/* <p>Result: {data}</p> */}
       </header>
-      <body>
-      </body>
+        {/* <form onSubmit={calculate}> */}
+        <form>
+          <label>first Number : <input name="firstValue" type="number" /></label>
+          <label>Operation : 
+            <select name="operation">
+              <option value={ Operations.Multiply }>Multiply</option>
```

Comment:
You could probably use the `Operations` values instead of the name, e.g. "*" rather than "Multiply"